app/services/core_data_connector/import/importer.rb

```diff
@@ -38,6 +35,12 @@ def initialize(directory)
 
           @importers << klass.new(filepath)
         end
+
+        # Import relationships last
```

Comment:
Ahh, good catch. I like that we're adding the relationships last. We're duplicating a little code where the file path is created, we're checking if the file exists, then instantiating the importer class. I think it would be more DRY to either:
1. Extract the above logic to its own method
2. Re-add the relationships importer to the `IMPORTERS` variable and add a comment that it must be last (was there a reason that we didn't do this? Not explicit enough?)